website/docs/r/is_lb_listener_policy.html.markdown

```diff
@@ -96,11 +126,15 @@ Review the argument references that you can specify for your resource.
 - `target_id` - (Optional, Integer) When `action` is set to **forward**, specify the ID of the load balancer pool that the load balancer forwards network traffic to.
 - `target_http_status_code` - (Optional, Integer) When `action` is set to **redirect**, specify the HTTP response code that must be returned in the redirect response. Supported values are `301`, `302`, `303`, `307`, and `308`. 
 - `target_url` - (Optional, Integer) When `action` is set to **redirect**, specify the URL that is used in the redirect response.
+- `taget_https_redirect_listener` - (Optional, String) When `action` is set to **https_redirect**, specify the ID of the listener that will be set as http redirect target.
+- `target_https_redirect_status_code` - (Optional, Integer) When `action` is set to **https_redirect**, specify the HTTP status code to be returned in the redirect response. Supported values are `301`, `302`, `303`, `307`, `308`.
+- `target_https_redirect_uri` - (Optional, String) When `action` is set to **https_redirect**, specify the target URI where traffic will be redirected.
 
 **Note**
 
 When action is `forward`, `target_id` should specify which pool the load balancer forwards the traffic to.
 When action is `redirect`, `target_url` should specify the `url` and `target_http_status_code` to specify the code used in the redirect response.
+When action is `https_redirect`, `taget_https_redirect_listener` should specify the ID of the listener, `target_https_redirect_status_code` to specify the code used in the redirect response and `target_https_redirect_uri` to specify the target URI where traffic will be redirected.
```

Comment:
should be * target_https_redirect_listener

Reply:
Done